src/analysis/excluded_terms_in_mondo/excluded_terms_in_mondo.py

```diff
@@ -0,0 +1,167 @@
+"""Excluded terms in / xrefed-in Mondo
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/31
+
+TODO's
+"""
+import os
+import sys
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+
+
+# Vars
+# # Static
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+# CACHE_DIR = os.path.join(THIS_SCRIPT_DIR, 'cache')
+ANALYSIS_DIR = os.path.join(THIS_SCRIPT_DIR, '..')
+PROJECT_DIR = os.path.join(ANALYSIS_DIR, '..', '..')
+MAPPINGS_DIR = os.path.join(THIS_SCRIPT_DIR, 'ignored', 'mappings')
+SIGNATURE_FILES_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology', 'reports')
+# # Config
+CONFIG = {
+    'ontologies': ['doid', 'icd10cm', 'icd10who', 'ncit', 'omim', 'ordo'],  # todo: get list from mondo config yml
+    # 'use_cache': False,
+    'mondo_mappings_path': os.path.join(MAPPINGS_DIR, 'mondo.sssom.tsv'),
```

Comment:
This is how I've parameterized the inputs/outputs of this script. If you want a redesign / refactor, just let me know.

Reply:
@matentzn Do you mind if I keep these defaults? I will change the `make` goal to pass args to this script, but it is helpful for me to have these defaults when I'm running in the debugger.

Reply:
Now that you mention it, you are correct. I'll make the update if I can fit it in w/ the timeframe, but if not I will make an issue to do later.

Reply:
done. rather than a test, I just pre-created a debug config. i can create more of those or a test to run them all when time